lib/__testfixtures__/get-set.input.js

```diff
@@ -0,0 +1,3 @@
+const arr = [1, 2, 3];
+arr.length = 5;
+const b = arr.length;
```

Comment:
So, `arr.length` should specifically not get rewritten. Keep this test, but the expectation should change. Also, add a similar test for string.length (with no setter).

To make the test pass (i.e. not modify those .lengths), add a function called `isDataProperty` which takes some useful info (probably type and property name?). For now, hard-code it to return true for string and array `length` properties. Add a TODO in it to use a more robust mechanism. Then, if `isDataProperty()` returns true, do not perform the transformation.

Also, please include both a JS and web example. #26 has some. (Note: there are no good examples for JS setters, only web setters. That's fine.)

Reply:
For arr.length not being rewritten, in the case of something like:
```
arr.length = 5;
```
Should I keep it as is? Or should I change it to:
```
Reflect_apply(Array_length_get, arr) = 5;
```

I currently have it as the former, but could easily switch to the latter

Reply:
You should keep it as is. You cannot assign to the result of a function.

Reply:
Yeah that’s what I expected, just wanted to be sure